spec/fixtures/invalid-typechecked/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rules": {
+    "no-unused-variable": true
```

Comment:
Thanks! Is that still true as of TSLint 5.x (which `linter-tslint` [depends on](https://github.com/AtomLinter/linter-tslint/blob/master/package.json#L45))? From the [changelog](https://github.com/palantir/tslint/blob/d8db2467b745513d3a3d02b197f46009cd685d81/CHANGELOG.md#api-breaks-for-custom-rules):
> This means that some rules that previously worked without the type checker now require it. This includes:
> * no-unused-variable